.gitignore

```diff
@@ -7,3 +7,4 @@ config/*.js
 logs
 .env
 .DS_Store
+.idea/
```

Comment:
We should a blank line at the end of this and other files that github is reporting